src/api/.rubocop.yml

```diff
@@ -1,6 +1,7 @@
 inherit_from: .rubocop_todo.yml
 
 AllCops:
+  TargetRubyVersion: 2.4.1
```

Comment:
@evanrolfe sorry, we are installing Ruby 2.4.0, so this should be `TargetRubyVersion: 2.4`

Can this and the regeneration of the todo file be in a different commit?